org/lateralgm/components/ActionList.java

```diff
@@ -150,7 +146,7 @@ public void save()
 	 * @return The frame opened or <code>null</code> if no
 	 * frame was opened.
 	 */
-	public static MDIFrame openActionFrame(MDIFrame parent, Action a)
+	public MDIFrame openActionFrame(MDIFrame parent, Action a)
```

Comment:
@JoshDreamland Please take two seconds to actually think about it.

Let me break it down for you how it is in 16b4:
1) FRAMES is a static map
2) The static method openActionFrame will make sure that FRAMES is a static list of every open action frame.
3) ActionList.save() is not static.
4) ActionList.save() loops through FRAMES and calls save() on each ActionFrame.
5) Say you have a project with two open object frames, with an action list and an open action frame on each. You save the project, both action lists will have save() called, each save() will save every open action frame. Instead of saving each frame once, it saves each frame as many times as there are action lists.

Do you understand now what the original issue was?